src/main/java/com/beerjournal/datamodel/entity/UserCollectionEntity.java

```diff
@@ -0,0 +1,88 @@
+package com.beerjournal.datamodel.entity;
+
+import java.util.Collection;
+import java.util.LinkedList;
+import java.util.Objects;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+public class UserCollectionEntity {
+
+	private Optional<String> id = Optional.empty();
+	private String userID;
```

Comment:
owszem, ale jeżeli mamy case że obiekty mamy na bazie a użytkownik z nich składa sobie kolekcję, to musiałbym robić usunięcie pojedynczych obiektów, wkładać je do kolekcji i persystować, co też jest średnim rozwiązaniem